modules/docs/arrow-docs/docs/docs/arrow-core/quickstart/linting/README.md

```diff
@@ -10,24 +10,24 @@ Some linters might complain about some code practices that are common when worki
 
 ### Working with ArrowFx
 
-When working with [Arrow Fx]({{ '/docs/fx' | relative_url }}) side effects are expressed as `suspend` functions. 
+When working with [Arrow Fx]({{ '/docs/effects/fx' | relative_url }}), side effects are expressed as `suspend` functions.
```

Comment:
```suggestion
When working with [Arrow Fx]({{ '/docs/fx' | relative_url }}), side effects are expressed as `suspend` functions.
```

Because when browsing to '/docs/effects/fx' I saw the previous design.